README.md

```diff
@@ -96,8 +96,8 @@ Then on each step of the training loop:
 
 ```rust
 extern crate tch;
-use tch::nn;
-use tch::Tensor;
+use tch::nn::*;
+use tch::*;
```

Comment:
Maybe we could avoid the wildcard uses as they bring lots of things in the scope and instead a `use tch::kind` would be enough?